test/NuGet.Clients.Tests/NuGet.SolutionRestoreManager.Test/NuGet.SolutionRestoreManager.Test.csproj

```diff
@@ -24,16 +24,7 @@
     <ProjectReference Include="..\..\TestUtilities\Test.Utility\Test.Utility.csproj" />
   </ItemGroup>
 
-  <ItemGroup>
-    <Reference Include="Microsoft.VisualStudio.Shell.Interop.15.3.DesignTime">
-      <HintPath>$(SolutionPackagesFolder)Microsoft.VisualStudio.Shell.Interop.15.3.DesignTime.15.0.26612\lib\Microsoft.VisualStudio.Shell.Interop.15.3.DesignTime.dll</HintPath>
-    </Reference>
-  </ItemGroup>
-
-  <ItemGroup>
-      <PackageReference Include="Microsoft.VisualStudio.Shell.Interop.15.3.DesignTime" Version="15.0.26606" ExcludeAssets="all" />
-  </ItemGroup>
-
   <Import Project="$(BuildCommonDirectory)common.targets"/>
+  <Import Project="$(BuildCommonDirectory)fixinterop.targets"/>
```

Comment:
where is this target defined?

Reply:
previous PR: 
https://github.com/NuGet/NuGet.Client/pull/2563/files#diff-a34366c2eb47a16cae969ef67d7ebf2a